flink-connectors/flink-connector-mongodb/src/test/java/org/apache/flink/mongodb/streaming/utils/ListSink.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.flink.mongodb.streaming.utils;
+
+import org.apache.flink.streaming.api.functions.sink.SinkFunction;
+
+import com.google.common.collect.Lists;
+
+import java.util.List;
+
+/**
+ * Sink that collects elements into a collection.
+ **/
+public class ListSink<IN> implements SinkFunction<IN> {
```

Comment:
More than creating this custom class, can you use `org.apache.flink.streaming.util.StreamCollector`?